core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -570,12 +725,39 @@ export class CreateServiceComponent implements OnInit {
         payload["events"] = [];
         payload["events"].push(event);
       }
+    }
+    else if(this.typeOfPlatform === 'azure') {
+      if(this.azureEventExpression.type !== "azureEventsNone") {
+        var event = {};
+        event["type"] = this.azureEventExpression.type;
+        if(this.azureEventExpression.type === "cosmosdb") {
+          event["source"] =  "subscriptions/fd312e60-798b-4933-a4c9-66fa2697a464/providers/Microsoft.cosmosdb/"+ this.model.domainName + "/" + this.azureEventExpression.cosmosdb;
```

Comment:
Nit: Would it not be clearer to extract the prefix "subscriptions/fd312e60-798b-4933-a4c9-66fa2697a464/providers/Microsoft." as a constant and then append the suffix?

Reply:
Wait, we can't have this hardcoded. Subscription id should come from config.

Reply:
Changed hardcoded values and fetching from config